packages/sites/src/pages/update-page.ts

```diff
@@ -23,9 +23,18 @@ import { removeUnusedResources } from "../layout/remove-unused-resources";
  */
 export function updatePage(
   model: IModel,
-  patchList: string[],
-  requestOptions: IHubRequestOptions
+  maybePatchList: string[] | Record<string, any>,
+  maybeRequestOptions?: IHubRequestOptions
 ): Promise<IUpdateItemResponse> {
+  let patchList = maybePatchList as string[];
+  let requestOptions = maybeRequestOptions;
+
+  // support old call signature
```

Comment:
this is the _new_ signature (where the second arg potentially _has_ a `.patchList` as opposed it _being_ a patch list).  It's really important to be clear in the comments here and in the names/docs of the args what will be supported going forward and what will be dropped at the next breaking change.